modules/core/02-client/keeper/grpc_query_test.go

```diff
@@ -116,13 +116,15 @@ func (suite *KeeperTestSuite) TestQueryClientStates() {
 		msg      string
 		malleate func()
 		expErr   error
+		expTotal int
```

Comment:
The name is a little confusing, since what you are checking is actually the count. So maybe this should be renamed to expCount.

Reply:
Also, please move this up to the beginning of the function (similar to `expClientStates`) and update it in malleate.